qiskit_ibm_runtime/quantum_program/quantum_program.py

```diff
@@ -137,7 +137,7 @@ def __init__(
         circuit: QuantumCircuit,
         samplex: Samplex,
         *,
-        samplex_arguments: dict[str, np.ndarray | PauliLindbladMap] | None = None,
+        samplex_arguments: dict[str, Any] | None = None,
```

Comment:
`np.ndarray | PauliLindbladMap` is wrong, the correct type for the dictionary values is `np.ndarray | dict[str, PauliLindbladMap]`. I prefer `Any`, which is less cumbersome, and also in order no to restrict types of new arguments that we may add in the future.